esp-wifi/src/esp_now/mod.rs

```diff
@@ -630,6 +687,8 @@ impl<'d> EspNow<'d> {
         Ok(esp_now)
     }
 
+    /// Splits the ESP-NOW instance into its manager, sender, and receiver
```

Comment:
```suggestion
    /// Splits the `EspNow` instance into its manager, sender, and receiver
```

I guess we should be consistent how we refer to the technology vs the implementation.